deploy/runtime/versions.lk

```diff
@@ -63,6 +63,7 @@ version_upgrade{'2021.7.27'} = define() {
 	old_temp_celsius = oldvalue('batt_room_temperature_celsius');
 	new_temp_vector = [];
 	if (is_batt_tech) {
+		message('The battery replacement and lifetime inputs are now on the new Battery Life input page. Battery life inputs change to the appropriate defaults when you change the battery type on the Battery Cell and System page and accept the default values.');
```

Comment:
I think the versions.lk script needs to include the variable "batt_life_model" as well. My preference would be to keep upgraded files with the same settings they had prior to the upgrade (cycle-life degradation for everything, I think) and have users to actively choose to use one of the new models.

Reply:
Good catch on this and the fuel cell behavior! Commit 29b48d25b should fix setting batt_life_model in the versions.lk script.

Commits f471ff0c1 and f3e60f26c should fix the fuel cell model using the old Battery Lifetime form because the Enable Battery choice requires that all battery inputs be on one page.